protocol/x/clob/keeper/process_operations.go

```diff
@@ -566,6 +581,18 @@ func (k Keeper) PersistMatchLiquidationToState(
 				),
 			)
 		}
+
+		filledQuoteQuantums := new(big.Int).Mul(
+			new(big.Int).Div(matchWithOrders.FillAmount.ToBigInt(), new(big.Int).SetUint64(matchLiquidation.TotalSize)),
```

Comment:
For my understanding, isn't `matchWithOrders.FillAmount <= matchLiquidation.TotalSize` always true?

Due to integer division rounding, does that mean this number will be either 0 or 1?

Reply:
Yes thank you, I should be using Quo not Div

Reply:
good callout, updating to use big.Rat and Quo to get the proper result here:
```
fillAmountToTotalSizeRat := new(big.Rat).Quo(
			new(big.Rat).SetUint64(matchWithOrders.FillAmount.ToUint64()),
			new(big.Rat).SetUint64(matchLiquidation.TotalSize),
		)
		filledQuoteQuantums := lib.BigRatRound(
			new(big.Rat).Mul(
				fillAmountToTotalSizeRat,
				new(big.Rat).SetInt(notionalQuoteQuantums),
			),
			true,
		)
		telemetry.IncrCounterWithLabels(
			[]string{types.ModuleName, metrics.LiquidationOrderNotionalQuoteQuantums, metrics.Filled, metrics.DeliverTx},
			float32(filledQuoteQuantums.Uint64()),
			matchLiquidation.GetMetricLabels(),
		)
```